### ISSUE TITLE
SyntaxError in `npx svg-to-ts --conversionType files`

### ISSUE THREAD

**Issue by kaigothe** [kreuzerk/svg-to-ts#138]
I try using svg-to-ts via npx: `$ npx svg-to-ts --conversionType files -s ./icons -o ./iconDist`

I got following output: 
```

==========================================================
👷 Hi I am svg-to-ts - let's build an awesome icon library
==========================================================
svg-to-ts: ℹ️ We are using the conversion type "files"
svg-to-ts: ✔ Deleting the output folder
svg-to-ts: ✔ Processing SVG files
svg-to-ts: ✔ Generate SVG constants
svg-to-ts: ✔ Generate Type Helper
svg-to-ts: ✔ Generate barrel file
svg-to-ts: ✖ Generate model file
svg-to-ts: SyntaxError: Type expected. (3:27)
  1 | /* 🤖 this file was generated by svg-to-ts*/
  2 | 
> 3 |     export type myIcons = ;export interface MyIcon{
    |                           ^
  4 |         name: myIcons;
  5 |         data: string;}

```

node version: v16.13.1
ppm version: 8.1.2

**Comment by kreuzerk** [comment]
Hiu @kaigothe I just tried your command in my project and everything works find. Can you please run `npx svg-to-ts --version` and make sure that you have the latest version of `svg-to-ts`.

**Comment by kaigothe** [comment]
Aloha, 
current version is:

```
$ npx svg-to-ts --version
7.1.0
```

**Comment by kreuzerk** [comment]
@kaigothe thx a lot for the reply. I will take another look but so far I can not reproduce this.

**Comment by kreuzerk** [comment]
Hi @kaigothe I investigated again and I think this error occurs if no `srcFiles` are found. I think the library should fail early if no sources are found. This is currently not the case. I added a story to improve this behavior.

I will let you know once I have improved this behavior. In the meantime, you can also try to run the command with the `--verbose` flag to get a more detailed log output.

**Comment by kreuzerk** [comment]
Can you retry again with version 8.2?

**Comment by Garbee** [comment]
I just got the chance to test with what we are doing at work with 8.4.0. It is now failing early with a proper error message.

At least in my case here, this is fixed.

Still can't generate an enum off of Material Icons. But I have a feeling that is an auxiliary problem due to source file naming not being 1-to-1 convertible into TypeScript. (Like I had to filter out some numeric filenames just to get it to work with a baseline.)
